packages/components/ng-uirouter-layout/src/index.js

```diff
@@ -69,10 +70,19 @@ angular
         }
 
         if (get(state, 'layout.name') === 'modal') {
+          let componentName = get(state, 'state.views.modal.component', state.component);
           const $state = transition.injector().get('$state');
           const $uibModal = transition.injector().get('$uibModal');
 
-          const componentName = get(state, state.views.modal ? 'views.modal.component' : 'component');
+          const componentProvider = get(state, 'state.views.modal.componentProvider', state.componentProvider);
```

Comment:
Theses changes are addressed in another PR https://github.com/ovh-ux/manager/pull/931 ?

Reply:
Oopsie ! Probably due to rebase, fixing it right away